councilmatic_core/management/commands/import_data.py

```diff
@@ -204,6 +212,9 @@ def organizations_etl(self,
             self.add_new_organizations()
             self.add_new_posts()
 
+        if clear_downloads:
+            shutil.rmtree(self.organizations_folder)
+
```

Comment:
@reginafcompton Are you planning to add this to the other download folders, too?

Reply:
+1 to this question, otherwise looks regular to me!

Reply:
oh, i just saw your comment in the description! fire away with the other folders.